Code/GraphMol/Depictor/Wrap/rdDepictor.cpp

```diff
@@ -93,12 +99,46 @@ unsigned int Compute2DCoordsMimicDistmat(
   }
   return res;
 }
+
+  void GenerateDepictionMatching2DStructure( RDKit::ROMol &mol ,
+					     RDKit::ROMol &reference ,
+					     int confId ,
+					     python::object refPatt ,
+					     bool acceptFailure ) {
+
+    RDKit::ROMol *referencePattern = 0;
+    if( refPatt ) {
+      referencePattern = python::extract<RDKit::ROMol *>( refPatt );
+    }
+    
+    RDDepict::generateDepictionMatching2DStructure( mol , reference ,
+						    confId , referencePattern ,
+						    acceptFailure );
+  }
+
+  void GenerateDepictionMatching3DStructure( RDKit::ROMol &mol ,
+					     RDKit::ROMol &reference ,
+					     int confId ,
+					     python::object refPatt ,
+					     bool acceptFailure ) {
+
+    RDKit::ROMol *referencePattern = 0;
+    if( refPatt ) {
```

Comment:
This will need to be changed to `refPatt != python::object()` once my suggested default argument change is made (see below)